hadoop.security.group.mapping.ldap.num.attempts	3	ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-2040322260, effective=null
		30